examples/notebooks/Poisson_non_periodic.ipynb

```diff
@@ -276,7 +282,7 @@
     "\n",
     "Pm = PostProcessManager(\n",
     "    domain=Omega,\n",
-    "    space_file=f'results_poisson/space_info_{Omega.name}.yml',\n",
+    "    space_file=f'results_poisson/space_info_{Omega.name}.yaml',\n",
```

Comment:
And maybe this line should be kept too. Was the output file name extension changed from `.yml` to `.yaml` at some point?